Editor/Utils/PackageManager/PackageManagerEventListener.cs

```diff
@@ -1,28 +1,32 @@
 ﻿using System.Linq;
 using ReadyPlayerMe.Core.Analytics;
-using ReadyPlayerMe.Core.Editor.Models;
 using UnityEditor;
 using UnityEditor.PackageManager;
+
 namespace ReadyPlayerMe.Core.Editor
 {
-    public class PackageManagerEventListener
+    public abstract class PackageManagerEventListener
     {
+        private const string REQUIRED_KEYWORD = "readyplayerme";
+        
         [InitializeOnLoadMethod]
         static void Initialize()
         {
             Events.registeringPackages += OnPackagesInstalled;
         }
+        
+        ~PackageManagerEventListener()
+        {
+            Events.registeringPackages -= OnPackagesInstalled;
+        }
 
         static void OnPackagesInstalled(PackageRegistrationEventArgs packageRegistrationEventArgs)
         {
             packageRegistrationEventArgs.added
-                .Select(package => new PackageCoreInfo
-                {
-                    Id = package.packageId,
-                    Name = package.displayName,
-                })
+                .Where(packageInfo => packageInfo.packageId.Contains(REQUIRED_KEYWORD))
```

Comment:
but then it will be fired for any package installed doesn't matter if its our partners or random 3rd party package which could cause a clutter in analytics events.

Reply:
Though if that's the intention I can remove this check.

Reply:
Do we need to worry about user permission in regards to tracking every package a developer installs? 😅

Reply:
It's not sensitive data, since we're just checking the package name, so I don't see why we would have a user permission problem - it's only if they opt into analytics which I believe is enough 🤔

> but then it will be fired for any package installed doesn't matter if its our partners or random 3rd party package which could cause a clutter in analytics events.

This could be super valuable though, maybe there's a package that's often used with RPM which we have no idea about. If so then we could reach out to them and maybe add them to the integrations page.